central/graphql/resolvers/node_vulnerabilities.go

```diff
@@ -47,19 +51,63 @@ func (resolver *Resolver) NodeVulnerability(ctx context.Context, args IDQuery) (
 	if !features.PostgresDatastore.Enabled() {
 		return resolver.nodeVulnerabilityV2(ctx, args)
 	}
-	// TODO : Add postgres support
-	return nil, errors.New("Resolver NodeVulnerability does not support postgres yet")
+
+	if err := readNodes(ctx); err != nil {
+		return nil, err
+	}
+
+	vulnLoader, err := loaders.GetNodeCVELoader(ctx)
+	if err != nil {
+		return nil, err
+	}
+	vuln, err := vulnLoader.FromID(ctx, string(*args.ID))
+	if err != nil {
+		return nil, err
+	}
+	vulnResolver, err := resolver.wrapNodeCVE(vuln, true, nil)
+	if err != nil {
+		return nil, err
+	}
+	vulnResolver.ctx = ctx
+	return vulnResolver, nil
 }
 
 // NodeVulnerabilities resolves a set of vulnerabilities based on a query.
-func (resolver *Resolver) NodeVulnerabilities(ctx context.Context, q PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
+func (resolver *Resolver) NodeVulnerabilities(ctx context.Context, args PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
```

Comment:
This file should have metrics added to the subresolver functions, can be addressed in a future PR just to get this one merged in